content/docs/security/cors.md

```diff
@@ -140,13 +220,31 @@ Use the {{< reuse "docs/snippets/k8s-gateway-api-name.md" >}} CRDs to create COR
    < x-envoy-upstream-service-time: 7
    < access-control-allow-origin: https://example.com/
    < access-control-allow-credentials: true
-   < access-control-expose-headers: origin, vh-header, route-header
+   < access-control-expose-headers: Origin, X-HTTPRoute-Header
    < server: envoy
    < 
    [{"id":1,"name":"Dog","status":"available"},{"id":2,"name":"Cat","status":"pending"}]
    ```
-   
-3. Send another request to the Petstore app. This time, you use `notallowed.com` as your origin. Although the request succeeds, you do not get back any CORS headers, because `notallowed.com` is not configured as a supported origin.  
+   {{% /tab %}}
+   {{% tab  %}}
+   ```console {hl_lines=[7,8,9]}
+   * Mark bundle as not supporting multiuse
+   < HTTP/1.1 200 OK
+   < content-type: text/xml
+   < date: Mon, 03 Jun 2024 17:05:31 GMT
+   < content-length: 86
+   < x-envoy-upstream-service-time: 7
+   < access-control-allow-origin: https://example.com/
+   < access-control-allow-credentials: true
+   < access-control-expose-headers: Origin, X-TrafficPolicy-Header
+   < server: envoy
+   < 
+   [{"id":1,"name":"Dog","status":"available"},{"id":2,"name":"Cat","status":"pending"}]
+   ```
+   {{% /tab %}}
+   {{< /tabs >}}
+
+2. Send another request to the Petstore app. This time, you use `notallowed.com` as your origin. Although the request succeeds, you do not get back any CORS headers, because `notallowed.com` is not configured as a supported origin.  
```

Comment:
I think that if you'll try `b.example.com` here you will get the same response like the notallowed although you added `*.example.com` to the allow origins. As I mentioned previously.